### ISSUE TITLE
Provide a Grafana dashboard for Hystrix Metrics

### ISSUE THREAD

**Issue by marcingrzejszczak** [micrometer-metrics/micrometer#1626]

**Comment by shakuzen** [comment]
@checketts I think you had some kind of dashboard for Hystrix metrics, if I remember correctly. Is any of that open source, or could it be?

**Comment by checketts** [comment]
I had one at a previous employer. But those were based of Hystrix Prometheus before Micrometer had support. So this Dashboard would match that: https://grafana.com/grafana/dashboards/2113

**Comment by marcingrzejszczak** [comment]
No it's not a high priority at all. I thought that maybe there is one already. We are absolutely going towards Micrometer with Resilience4j

**Comment by AdamAntinoo** [comment]
I have created a SpringBoot simple hystrix command surrounding a test repository and afterwards collected the metrics. I have adapted the proposed Hystrix Dashboard and fixed most of the panels.
There is a set of panels that seems not compatible with current metrics because latency data is not being reported. This is my first effort and I wish to know if I should go ahead and rework the panels not compatible or just replace them by other useful data from Micrometer metrics.

I have attached the Grafana JSON dashboard definition.
[Hystrix Dashboard-1585394436235.json.txt](https://github.com/micrometer-metrics/micrometer/files/4397026/Hystrix.Dashboard-1585394436235.json.txt)
